lib/CL/clUnloadPlatformCompiler.c

```diff
@@ -22,11 +22,25 @@
  **/
 
 #include "pocl_cl.h"
+#include "pocl_llvm.h"
 
 CL_API_ENTRY cl_int CL_API_CALL
 POname(clUnloadPlatformCompiler)(cl_platform_id platform)
 CL_API_SUFFIX__VERSION_1_2
 {
+#if defined(BUILD_ICD) && defined(OCS_AVAILABLE)
```

Comment:
Unless i'm misreading the code in clGetPlatformIDs, calling clGetPlatformIDs() only returns a valid platform when BUILD_ICD is #defined. (which TBH might be a bug)